### PR TITLE
integrated tether into core 

### PR DESCRIPTION
**Pull request by luna215**
# Checklist

- [ ] PR has the correct target milestone when it's merged
- [ ] Automated test coverage is satisfactory
- [ ] PR has been fully tested manually
- [ ] Documentation is updated as necessary
- [ ] External dependency files are updated (`yarn` and `pip`)
- [ ] If internal dependency are updated, link to diff is included
- [ ] Screenshots of any front-end changes are in the PR description
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] You've added yourself to AUTHORS.rst if you're not there

# Details
Since tether is dependent on keen-ui, then we would have to pull an entire copy of tether anytime we would import it in any other module. By integrating tether and tether-x modules into the core API, we can reference them on other modules in Kolibri without importing an entire hard copy of the modules. Thus, optimizing the overall size of the project.

### Summary
Experimented with putting tether and tether-x modules into the core API.

* description of the change
 File(s) changed: kolibri/core/assets/src/core-app/apiSpec.js
* manual verification steps performed
  searched 
* screenshots if the PR affects the UI

### Reviewer guidance

description of how to test the changes

### References

when applicable, please provide:

* references to related issues and PRs
* links to mockups or specs for new features
* links to the diffs for dependency updates, e.g. in iceqube or the perseus plugin

### CODE REVIEW

**Comment by christianmemije** [comment]
@luna215 Could you document the motivation for this change and the improvements that result from this?

**Comment by luna215** [comment]
@christianmemije  I updated the **Details** to include the motivation and what improved based on the changes.

**Comment by rtibbles** [comment]
More details of the improvements can be seen here: https://docs.google.com/spreadsheets/d/1NueLUVF7WfOyEweq21bkhG4o6GBvf6AHWfZoLBl1T00/edit#gid=0

File: AUTHORS.rst [hunk 1]
```diff
@@ -26,3 +26,4 @@ Please feel free to add your name on this list if you do a PR!
 * Alan Chen (alanchenz)
 * Yixuan Liu (yil039)
 * Gerardo Soto (gcodeon)
+* Paul Luna
```
Comment (rtibbles) [inline AUTHORS.rst]: @luna215 can you add your github username in parentheses here? (see the examples above!)

**Pull request merged**